protoc-php.php

```diff
@@ -72,13 +76,14 @@
         printf('  -p, --filename-prefix [PREFIX]    Specify a prefix for generated file names' . PHP_EOL);
         printf('  -t [path]                         Target directory for output' . PHP_EOL);
         printf('  --psr                             Output class files in a psr-4 directory structure' . PHP_EOL);
+        printf('  --compile-in-place                Generated files will be outputted in the same directory as their source files' . PHP_EOL);
         exit(1);
     }
 
     // Reindex argv
     $GLOBALS['argv'] = array_values($GLOBALS['argv']);
 
-    $parser = new ProtobufParser($useNamespaces);
+    $parser = new ProtobufParser($useNamespaces, $shouldCompileInPlace);
```

Comment:
this would potentially be more consistent with the way opts are set below

```
     if ($shouldCompileInPlace !== false) {
          $parser->setShouldCompileInPlace(true);
     }
```